### PR TITLE
prevent default  wont fire if keypress is backspace

### PR DESCRIPTION
**Pull request by alexalala**
https://trello.com/c/j9EoVMY8/238-bug-in-firefox-cannot-use-backspace-key-in-the-number-part-of-the-voucher-input-box

🐖

### CODE REVIEW

File: src/pages/Tap.vue [hunk 1]
```diff
@@ -162,15 +162,17 @@ export default {
         onKeypressVoucherBox : function(event) {
             var rxNumber = /\d/;
             var char = this.getKeyCharCode(event);
-
-            if (char.match(rxNumber)) {
-                if (this.voucherCode.length < event.target.maxlength) {
-                    this.voucherCode += char;
+            
+            if (event.keyCode != 8) {
```
Comment (katjam) [inline src/pages/Tap.vue]: Comment that event.keycode 8 is backspace

and use !== to be typesafe?

**Review by katjam — CHANGES_REQUESTED** [review]

File: src/pages/Tap.vue (hunk 1 above)
Comment (alexalala) [reply src/pages/Tap.vue]: @katjam ok!

**Review by katjam — APPROVED** [review]
:label: Good stuff.

**Review by charlesstrange2 — APPROVED** [review]
Neat!
:sheep::it:

**Pull request merged**